reservation.yaml.sample

```diff
@@ -24,6 +24,10 @@ role_distribution: debug
 #enable_monitoring: true
 #enable_rally: true
 
+# Enable for Nova to run in /tmp, allowing larger flavors
+# to be deployed
+enable_nova_bind: false
```

Comment:
Fixed in 22da28bcb662faa0ab38f5f78b7209da9ca64ed4

I also renamed the parameter to `enable_nova_tmp` which is (I think) more self-descriptive.